.env

```
@@ -4,3 +4,5 @@ VUE_APP_DESC='COVID-19 Sledilnik Slovenija - zbrani, preverjeni, ažurirani, ana
 VUE_APP_URL='https://covid-19.sledilnik.org'
 VUE_APP_DEFAULT_LANGUAGE='sl'
 VUE_APP_LOCALE_CONTEXT='SVN'
+VUE_APP_STRIPE_PUBLISHABLE_KEY='pk_test_51I6cVwEUtJJGJFIUgsLpXxbk4DHr2g8Q2ZCIUlzSWbVBtWbcEZJdJuRhyjCxO0xFZdBecREiHx2zirBDPFN1qoeQ00A5Wd3bak'
+VUE_APP_STRIPE_SUBSCRIPTIONS='5:price_1I6dUEEUtJJGJFIUM4slrZ9g;20:price_1I6dUEEUtJJGJFIURHozzi1V;50:price_1I6dUEEUtJJGJFIUnmagHiRR;STO:price_1I6dUFEUtJJGJFIUcWNRbSDC'
```

@overlordtm "STO" comes from config and is different in env.production

---

We could also change it:
```suggestion
VUE_APP_STRIPE_SUBSCRIPTIONS='5:price_1I6dUEEUtJJGJFIUM4slrZ9g;20:price_1I6dUEEUtJJGJFIURHozzi1V;50:price_1I6dUEEUtJJGJFIUnmagHiRR;100:price_1I6dUFEUtJJGJFIUcWNRbSDC'
```
but i wanted some visual indication that it differs from production config

---

synced with prod settings in https://github.com/sledilnik/website/pull/793/commits/eefbab1e906a72bd193afbbf564b0bad935b1c42